tests/test_solution.py

```diff
@@ -62,6 +62,22 @@ def test_solution_evaluation():
     assert sol[expr] == m.getVal(expr)
     assert sol[expr2] == m.getVal(expr2)
 
+def test_getSolTime():
+    m = Model()
+    m.setPresolve(SCIP_PARAMSETTING.OFF)
+
+    x = {}
+    for i in range(20):
+        x[i] = m.addVar(ub=i)
+
+    for i in range(1,6):
+        m.addCons(quicksum(x[j] for j in range(20) if j%i==0) >= i)
+        m.addCons(quickprod(x[j] for j in range(20) if j%i==0) <= i**3)
+    
+    m.setObjective(quicksum(x[i] for i in range(20)))
+    m.optimize()
+    for s in m.getSols():
+        m.getSolTime(s)
```

Comment:
There should be an assert here, to at least check that it's a positive value, and potentially check that each one has a time after the other, I'm assuming they are returned in the order they were found but I could be wrong here.

Reply:
Yeah, agreed. But unfortunately, it seems that ```getSols``` doesn't return solutions in the order they were found.